pkg/services/cloudmigration/model.go

```diff
@@ -162,12 +162,62 @@ type CloudMigrationSessionListResponse struct {
 	Sessions []CloudMigrationSessionResponse
 }
 
+type ResultSortColumn string
+
+const (
+	SortColumnID     ResultSortColumn = "id"
+	SortColumnName   ResultSortColumn = "name"
+	SortColumnType   ResultSortColumn = "resource_type"
+	SortColumnStatus ResultSortColumn = "status"
+)
+
+func (s ResultSortColumn) String() string {
+	switch s {
+	case SortColumnID:
+		return string(SortColumnID)
+	case SortColumnName:
+		return string(SortColumnName)
+	case "type":
+		fallthrough
+	case SortColumnType:
+		return string(SortColumnType)
+	case SortColumnStatus:
+		return string(SortColumnStatus)
+	default:
+		return string(SortColumnID)
+	}
+}
+
+type SortOrder string
+
+const (
+	SortOrderAsc  SortOrder = "ASC"
+	SortOrderDesc SortOrder = "DESC"
+)
+
+func (s SortOrder) String() string {
+	switch s {
+	case SortOrderAsc:
+		return string(SortOrderAsc)
+	case SortOrderDesc:
+		return string(SortOrderDesc)
+	default:
```

Comment:
Alternatively, it's a bit more annoying to use but implies that no `SortOrder` could hold an invalid value:
```go
func NewSortOrder(order string) (SortOrder, error) {
	so := SortOrder(so)
	if so != SortOrderAsc || so != SortOrderDesc {
		return "", errors.New("some error")
	}
	return so, nil 
}